automation/check-patch.packages.el7

```diff
@@ -2,13 +2,13 @@ ansible
 bats
 git
 libffi-devel
+libguestfs-devel
 libvirt-devel
 libxslt-devel
 openssl-devel
 pandoc
 python2-devel
 python-dulwich
-python-libguestfs
```

Comment:
Just to  be careful - I didn't want python-libguestfs to be available on the system for the unit testing. When we'll run the functional CLI tests, we install from RPM anyways so it will find its way to there.